iiblib/iib_build_details_model.py

```diff
@@ -1,135 +1,126 @@
 class IIBBuildDetailsModel(object):
     """Model class handling data about index build task"""
 
-    def __init__(
-        self,
-        _id,
-        state,
-        reason,
-        state_history,
-        from_index,
-        from_index_resolved,
-        bundles,
-        removed_operators,
-        organization,
-        binary_image,
-        binary_image_resolved,
-        index_image,
-        request_type,
-        arches,
-        bundle_mapping,
-        omps_operator_version,
-    ):
-        """
-        Args:
-            _id (int)
-                Id of build
-            state (str)
-                State of build
-            state (str)
-                Reason for state change
-            from_index (str)
-                Reference of index image used as source for rebuild
-            from_index_resolved (str)
-                Reference of new index image
-            bundles (list)
-                List of bundles to be added to index image
-            removed_operators (list)
-                List of operators to be removed from index image
-            organization (str)
-                Name of organization to push to in the legacy app registry
-            binary_image (str)
-                Reference of binary image used for rebuilding
-            binary_image_resolved (str)
-                Checksum reference of binary image that was used for rebuilding
-            index_image (str)
-                Reference of index image to rebuild
-            request_type (str)
-                Type of iib build task (add or remove)
-            arches (list)
-                List of architectures supported in new index image
-            bundle_mapping (dict)
-                Operator names in "bundles" map to: list of "bundles" which
-                map to the operator key
-            omps_operator_version (dict)
-                Operator version returned from OMPS API call used for Add request
-        """
-        self.id = _id
-        self.state = state
-        self.reason = reason
-        self.state_history = state_history
-        self.from_index = from_index
-        self.from_index_resolved = from_index_resolved
-        self.bundles = bundles
-        self.removed_operators = removed_operators
-        self.organization = organization
-        self.binary_image = binary_image
-        self.binary_image_resolved = binary_image_resolved
-        self.index_image = index_image
-        self.request_type = request_type
-        self.arches = arches
-        self.bundle_mapping = bundle_mapping
-        self.omps_operator_version = omps_operator_version
+    _data_attrs = []
+    _kwattrs = []
+
+    def __init__(self, *args, **kwargs):
+        self.data = {}
+        for attr, attr_val in zip(args, self._data_attrs):
```

Comment:
Shouldn't the names of attr and attr_val be switched?

Reply:
Yeah, but also the line below should be updated as well.
`self.data[attr] = attr_val`

Reply:
resolved